.github/workflows/bnetlauncher-build.yaml

```diff
@@ -1,6 +1,6 @@
 name: Build bnetlauncher
 
-on: [push]
+on: workflow_dispatch
```

Comment:
Why are you changing the pipeline to manual only?

Reply:
Sorry. I changed this to build myself

Reply:
on can be a list so `[push, workflow_dispatch]` would be valid.